core/src/smartcontracts/isi/permissions/judge.rs

```diff
@@ -45,6 +45,12 @@ pub trait Judge {
         wsv: &WorldStateView,
     ) -> Result<()>;
 
+    /// Disable operation description in validation error message
+    ///
+    /// Useful when one judge is nested in the other judge and we don't want to
+    /// print operation description twice in the error message
+    fn no_display_operation(&mut self);
```

Comment:
```suggestion
    /// Disable showing the operation description in the validation error message
    ///
    /// Use this when you have one [`Judge`] nested inside another, as it will prevent 
    /// printing the same error message twice. 
    fn disable_display_of_operation_on_error(&mut self);
```